src/loki_alert_rules/error_level_logs_found.rule

```diff
@@ -0,0 +1,7 @@
+alert: ErrorLevelLogsFound
+expr: rate({%%juju_topology%%} |= "level=error"[1m]) > 0
```

Comment:
Unlike Prometheus alert rules, in Loki based alert rules `%%juju_topology%%` is needed to [be replaced](https://github.com/canonical/loki-k8s-operator/blob/main/lib/charms/loki_k8s/v0/loki_push_api.py#L768):

```python
alert_rule["expr"] = self.tool.inject_label_matchers(
                            re.sub(r"%%juju_topology%%", r'job=".+"', alert_rule["expr"]),
                            self.topology.label_matcher_dict,
                        )
``` 

At this point this `exp` will be:

```
rate({job=".+"} |= "level=error"[1m]) > 0
```

After that juju_topology is added using `cos-tool`, so the `expr` will be:

```
(rate({job=".+", juju_application="traefik", juju_charm="traefik-k8s", juju_model="cos",
      juju_model_uuid="381236d6-8c63-4415-80ad-bfdccc1ddb4c"} |= "level=error"[1m])
      > 0)
```